docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,103 @@
+   :::
+
+### JSON editor features
+
+The JSON editor is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/). The Monaco Editor is the editor that powers VS Code. As a result, the template editor supports many familiar features such as auto-formatting, indentation support, code completion, and error highlighting. Read the [VS Code editor docs](https://code.visualstudio.com/docs/editor/editingevolved) for a full overview of features.
+
+### Versioning connector templates
+
+You can version your connector templates using [milestones](/components/modeler/web-modeler/milestones.md), similar to diagrams.
+
+## Fixing problems in your templates {#fixing-template-problems}
+
+While working an a template, the template will be in invalid intermediate states. For instance, when you add a new property, it must contain various mandatory attributes. Unless all mandotary attributes are defined, the template will not be saved, and the preview is not updated. This ensures that you can never publish an invalid or broken template.
+
+The editor toolbar indicates if the template is currently in a valid state or not. The JSON editor provides you with error highlighting, allowing you to add mandatory values and resolve problems without missing anything.
+
+![Indicating problems in connector templates](img/connector-templates/fix-connector-template-problems.png)
+
+If there are problems at the root level of the JSON (such as a missing or misspelled mandatory attribute), the error is highlighted in the first line of the editor. You need to click the error marker at the curly bracket to expand the error message.
+
+![Some connector template problems highlighted in the first line](img/connector-templates/fix-connector-template-problems-2.png)
+
+<!--
+## Reusing existing runtimes
+
+Not only can you create new connector templates to bind to a job worker topic or a custom connector SDK function, but you can also reuse [existing connector runtimes](/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md). For instance, if you add the following property and bind it to `zeebe:taskDefinition:type`, you can reuse the HTTP JSON runtime and write a prefilled connector template to call an API easily.
+
+```json
+{
+    "type": "Hidden",
+    "value": "io.camunda:http-json:1",
+    "binding": {
+        // highlight-next-line
+        "type": "zeebe:taskDefinition:type"
+    }
+}
+```
+-->
+
+## Importing existing connector templates
+
+If you have created templates for Desktop Modeler in the past and want to reuse them in Web Modeler, you need to make some adjustments to the template files.
+
+1. _Split the files_. Web Modeler maintains a 1:1 relation between connector templates and files. Since Desktop Modeler allows to keep multiple template definitions in single file, you must split the file in advance to one file per temmplate before uploading.
```

Comment:
```suggestion
1. _Split the files_. Web Modeler maintains a 1:1 relation between connector templates and files. Since Desktop Modeler allows to keep multiple template definitions in single file, you must split the file in advance to one file per template before uploading.
```